test/lib/usd/translators/testUsdExportTypes.py

```diff
@@ -0,0 +1,123 @@
+        cmds.file(self.testFile, force=True, open=True)
+        usdFile = os.path.abspath('UsdExportMeshOnly.usda')
+
+        cmds.usdExport(mergeTransformAndShape=True, file=usdFile,
+            shadingMode='none', excludeExportTypes=['Cameras','Lights'])
+        stage = Usd.Stage.Open(usdFile)
+    
+        meshPrims = ['/pSphere1','/pCube1']
+        for m in meshPrims:
+            meshPrim = stage.GetPrimAtPath(m)
+            self.assertTrue(meshPrim)
+
+    def _ValidateLight(self):
+        cmds.file(self.testFile, force=True, open=True)
+        usdFile = os.path.abspath('UsdExportLightOnly.usda')
+
+        cmds.usdExport(mergeTransformAndShape=True, file=usdFile,
+            shadingMode='none', excludeExportTypes=['Cameras','Meshes'])
+        stage = Usd.Stage.Open(usdFile)
+    
+        lightPrims = ['/directionalLight1','/pointLight1','/areaLight1','/spotLight1']
+        for l in lightPrims:
+            lightPrim = stage.GetPrimAtPath(l)
+            self.assertTrue(lightPrim)
+
+    def _ValidateCamera(self):
+        cmds.file(self.testFile, force=True, open=True)
+        usdFile = os.path.abspath('UsdExportCameraOnly.usda')
+
+        cmds.usdExport(mergeTransformAndShape=True, file=usdFile,
+            shadingMode='none', excludeExportTypes=['Lights','Meshes'])
+        stage = Usd.Stage.Open(usdFile)
+    
+        camera = '/camera1'
+        cameraPrim = stage.GetPrimAtPath(camera)
+        self.assertTrue(cameraPrim)
+
+    def _ValidateMeshCamera(self):
+        cmds.file(self.testFile, force=True, open=True)
+        usdFile = os.path.abspath('UsdExportMeshCamera.usda')
+
+        cmds.usdExport(mergeTransformAndShape=True, file=usdFile,
+            shadingMode='none', excludeExportTypes=['Lights'])
+        stage = Usd.Stage.Open(usdFile)
+    
+        camera = '/camera1'
+        cameraPrim = stage.GetPrimAtPath(camera)
+        self.assertTrue(cameraPrim)
+
+        meshPrims = ['/pSphere1','/pCube1']
+        for m in meshPrims:
+            meshPrim = stage.GetPrimAtPath(m)
+            self.assertTrue(meshPrim)
+
+    def _ValidateMeshLight(self):
+        cmds.file(self.testFile, force=True, open=True)
+        usdFile = os.path.abspath('UsdExportMeshCamera.usda')
+
+        cmds.usdExport(mergeTransformAndShape=True, file=usdFile,
+            shadingMode='none', excludeExportTypes=['Cameras'])
+        stage = Usd.Stage.Open(usdFile)
+    
+        lightPrims = ['/directionalLight1','/pointLight1','/areaLight1','/spotLight1']
+        for l in lightPrims:
+            lightPrim = stage.GetPrimAtPath(l)
+            self.assertTrue(lightPrim)
+
+        meshPrims = ['/pSphere1','/pCube1']
+        for m in meshPrims:
+            meshPrim = stage.GetPrimAtPath(m)
+            self.assertTrue(meshPrim)
+    
+    def testExportTypes(self):
+        self._ValidateMesh()
+        self._ValidateLight()
+        self._ValidateCamera()
+        self._ValidateMeshCamera()
+        self._ValidateMeshLight()
```

Comment:
No need to test the case of export all three, that is already covered by other unit tests